libs/api-interfaces/src/lib/api-interfaces.ts

```diff
@@ -1,3 +1,41 @@
 export interface Message {
   message: string;
 }
+
+export interface Candidate {
+    id: number;
+    name: string;
+    age: number;
+    motherTounge: string;
+    education: 'Barneskole' | 'Ungdomsskole' | 'Videregående' | 'Høyskole/Universitet' | 'Ingen';
+    benefits?: string;
+    handicap?: string;
+    assistiveTechnology?: string;
+    digitalSkills: 'Dårlig' | 'Gjennomsnittelig' | 'Bra';
+    employed: boolean;
+    industry?: string;
+    traits?: Trait[];
+}
+
+export interface Trait {
+    id: number;
+    name: string;
+}
+
+export interface Consent {
+    id: number;
+    name: string;
+}
+
+type insightTechnique = 'Intervju' | 'Lydopptak' | 'Videoopptak';
+
+export interface Insight {
+    name: string;
+    insightTechnique: insightTechnique;
+    description?: string;
+    starts?: string;
+    ends?: string;
+    candidates?: Candidate[];
+    traits?: Trait[];
+    consents?: Consent[];
+}
```

Comment:
It would be preferable to have fewer nullable types on these interfaces, to make our code more secure and less prone to errors. Fine for testing, but should be discussed eventually. :)

Reply:
Agreed

Reply:
I have created an issue that we have to design the data structure for the application, we can discuss this in that issue :)